unittests/db_to_kv_tests.cpp

```diff
@@ -0,0 +1,309 @@
+#include <eosio/testing/tester.hpp>
+#include <eosio/chain/db_key_value_format.hpp>
+
+#include <boost/test/unit_test.hpp>
+#include <cstring>
+
+using key_type = eosio::chain::db_key_value_format::key_type;
+using name = eosio::chain::name;
+
+BOOST_AUTO_TEST_SUITE(db_to_kv_tests)
+
+const uint64_t overhead_size = sizeof(name) * 2 + 1; // 8 (scope) + 8 (table) + 1 (key type)
```

Comment:
Should we prefer constexpr over const here?

Reply:
We can, so I changed it.